ocaml/testsuite/tests/translprim/array_spec.heap.flat.reference

```diff
@@ -23,7 +23,7 @@
       (let
         (eta_gen_len =
            (function {nlocal = 0} prim[genarray] stub
-             (array.length[gen] prim))
+             ignore assert all zero_alloc : int (array.length[gen] prim))
```

Comment:
Why did these reappear? Do you know what code change caused it?

Reply:
There was some issue with the PR, that produces wrong output for this test case. I fixed that issue and the `.stack.flat.reference`, but forgot to update the `.heap.flat.reference` (only tested with `--disable-stack-allocation`), and left it in the wrong state. This is now fixed.